ansible/files/bootstrap/telemetry.sh

```diff
@@ -270,10 +285,10 @@ EOF
     sudo tee -a /etc/datadog-agent/datadog.yaml <<EOF
 
 tags:
-- cluster_name:${CLUSTER_NAME}
-- cluster_id:${CLUSTER_ID}
+- cluster-name:"${CLUSTER_NAME}"
+- cluster-id:"${CLUSTER_ID}"
```

Comment:
Datadog seems to consistently use kebab-case in its metrics naming, so I'm moving to that here.